src/wp-includes/class-wp-widget.php

```diff
@@ -598,7 +598,7 @@ public function _register_one( $number = -1 ) {
 	 */
 	public function save_settings( $settings ) {
 		$settings['_multiwidget'] = 1;
-		update_option( $this->option_name, $settings );
+		update_option( $this->option_name, $settings, wp_use_widgets_block_editor() );
```

Comment:
Per https://github.com/WordPress/wordpress-develop/pull/6465#discussion_r1731918565, please revert as this should be handled via https://core.trac.wordpress.org/ticket/42743.
```suggestion
		update_option( $this->option_name, $settings );
```